porting/npl/riot/src/nrf5x_isr.c

```diff
@@ -19,30 +19,30 @@
 
 #include "cpu.h"
 
-static uint32_t radio_isr_addr;
-static uint32_t rng_isr_addr;
-static uint32_t rtc0_isr_addr;
+static void (*radio_isr_addr)();
+static void (*rng_isr_addr)();
+static void (*rtc0_isr_addr)();
```

Comment:
Same here, RIOT complains about `[-Werror=strict-prototypes]`, so please add the missing `void`:
```c
static void (*radio_isr_addr)(void);
static void (*rng_isr_addr)(void);
static void (*rtc0_isr_addr)(void);
```